@xen-orchestra/web-core/lib/components/ui/alarm-item/UiAlarmItem.vue

```diff
@@ -0,0 +1,144 @@
+<!-- v1 -->
+<template>
+  <li class="ui-alarm-item" :class="className">
+    <div class="content">
+      <div class="label-value text-ellipsis">
+        <UiButtonIcon
+          v-if="alarm.description"
+          :icon="isDescriptionVisible ? faAngleDown : faAngleRight"
+          size="small"
+          accent="brand"
+          @click="toggleDescription()"
+        />
```

Comment:
As we use the button in its "small" size, the clickable area is too small, which is bad for accessibility.
The trick here is to use the prop `target-scale` and set a number that would make the clickable area bigger:
```suggestion
        <UiButtonIcon
          v-if="alarm.description"
          :icon="isDescriptionVisible ? faAngleDown : faAngleRight"
          :target-scale="2"
          size="small"
          accent="brand"
          @click="toggleDescription()"
        />
```
[Here](https://ishadeed.com/article/target-size/#extend-target-size-with-pseudo-elements) is an explanation on how it works.